ui/job-view/details/tabs/SimilarJobsTab.jsx

```diff
@@ -97,10 +97,18 @@ class SimilarJobsTab extends React.Component {
           this.showJobInfo(newSimilarJobs[0]);
         }
       } else {
-        notify(`Error fetching similar jobs push data: ${data}`, 'danger', {
-          sticky: true,
-        });
+        notify(
+          `Error fetching similar jobs push data: ${failureStatus}`,
```

Comment:
Using the `data` that's returned, rather than `failureStatus` here would display a clearer error message.